hebrew/fuck/your-sister.md

```diff
@@ -0,0 +1,5 @@
+איכ אחי.
+
+לך בפינה ותחשוב על מה שעשית.
+
+[חזור להתחלה ותקווה שהכל היה חלום רע](../מרשמלו.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.